scripts/breaking_changes_checker/breaking_changes_allowlist.py

```diff
@@ -8,7 +8,6 @@
 
 RUN_BREAKING_CHANGES_PACKAGES = [
     "azure-ai-formrecognizer",
-    "azure-ai-textanalytics"
```

Comment:
This legacy breaking changes checker is raising a false positive for `is_error` so skipping it in CI.